lighthouse/blueprints/cherrypicked_plates.py

```diff
@@ -28,14 +28,18 @@
 CORS(bp)
 
 
-@bp.route("/cherrypicked-plates/create", methods=["POST"])
+@bp.route("/cherrypicked-plates/create", methods=["GET"])
 def create_plate_from_barcode() -> Tuple[Dict[str, Any], int]:
+
+    invalid_url = {"errors": ["GET request needs 'barcode' in url"]}, HTTPStatus.BAD_REQUEST
     try:
-        barcode = request.get_json()["barcode"]
+        barcode = request.args.get('barcode', '')
+        if len(barcode) == 0:
+            return invalid_url
         logger.info(f"Attempting to create a plate in SS from barcode: {barcode}")
```

Comment:
Could we add a test that checks this error condition, when no barcode is provided?

Reply:
def test_get_cherrypicked_plates_endpoint_no_barcode_in_request
is testing if the barcode doesnt exist in the url then it will throw this error